src/app/progress/FactionIcon.tsx

```diff
@@ -25,7 +24,7 @@ export default function FactionIcon(props: {
           <polygon strokeDasharray="121.622368" style={style} fillOpacity="0" stroke="#FFF" strokeWidth="3" points="24,2.5 45.5,24 24,45.5 2.5,24" strokeLinecap="butt"/>
         }
       </svg>
-      <div className={classNames('item-stat', 'item-faction', { 'purchase-unlocked': factionProgress.level >= 10 })}>{factionProgress.level}</div>
+      <div className="item-stat item-faction">{factionProgress.level}</div>
```

Comment:
we should keep the "purchase unlocked" for the planetary vendors because they don't use the new "seasonal ranking"